catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -327,6 +350,7 @@
                             !org.apache.commons.cli.*,
                             !org.hibernate.*,
                             !weblogic.jdbc.*,
+                            !org.abego.treelayout.*,
```

Comment:
Still needed or the wiring fails in Karaf due to trying to find `org.abego.treelayout`